docs/Site-Reliability-Guardian/example-pipelines/AzureDevOps/yaml-pipelines.md

```diff
@@ -1,6 +1,6 @@
 # Azure DevOps integration using YAML pipelines
 
-To integrate the CLI with Azure DevOps you can use the yaml pipeline that references a docker container:
+To integrate the CLI with Azure DevOps you can use the `yaml` pipeline that references a docker container or use the [complete file here](./azdo.yaml)
```

Comment:
what does yaml pipeline mean? does it refer to a pipeline in yaml format?

Reply:
yes, i will adjust the wording